apps/fyle/views.py

```
@@ -266,7 +266,7 @@ def post(self, request, *args, **kwargs):
         """
         try:
             workspace = Workspace.objects.get(pk=kwargs['workspace_id'])
-            FyleCredential.objects.get(workspace_id=workspace.id)
+            fyle_credentials = FyleCredential.objects.get(workspace_id=workspace.id)
```

Ruff (0.8.2)

Local variable `fyle_credentials` is assigned to but never used

Remove assignment to unused variable `fyle_credentials`

(F841)